roles/tqm/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+#########################################################################
+# Title:            Sandbox: tqm                                        #
+# Author(s):        CHAIR/Raneydazed                                    #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: Check if 'tqm_service' exists
+  ansible.builtin.stat:
+    path: "{{ tqm_paths_service_location }}"
+  register: tqm_service
+
+- name: Stop existing 'tqm_service'
+  ansible.builtin.systemd:
+    state: stopped
+    name: tqm.service
+  when: tqm_service.stat.exists
+
+- name: Check if 'tqm_timer' exists
+  ansible.builtin.stat:
+    path: "{{ tqm_paths_timer_location }}"
+  register: tqm_timer
+
+- name: Stop existing 'tqm_timer'
+  ansible.builtin.systemd:
+    state: stopped
+    name: tqm.timer
+  when: tqm_timer.stat.exists
+
+- name: Create directories
+  ansible.builtin.include_tasks: "{{ resources_tasks_path }}/directories/create_directories.yml"
+
+- name: Set variable for output of tqm releases
+  ansible.builtin.uri:
+    url: https://api.github.com/repos/l3uddz/tqm/releases/latest
+  register: tqm_releases
+
+- name: Get the url for latest amd64 tqm binary
+  ansible.builtin.set_fact:
+    tqm_latest: "{{ tqm_releases.json.assets | selectattr('name', '==', 'tqm_v1.5.0_linux_amd64') | map(attribute='browser_download_url') | join('') }}"
+
+- name: Download tqm binary
+  ansible.builtin.get_url:
+    url: "{{ tqm_latest }}"
+    dest: "{{ tqm_paths_location }}/tqm"
+    owner: "{{ user.name }}"
+    group: "{{ user.name }}"
+    mode: "0775"
+    force: true
+
+- name: Check if `{{ tqm_paths_config_location | basename }}` exists
+  ansible.builtin.stat:
+    path: "{{ tqm_paths_config_location }}"
+  register: tqm_config
+
+- name: "Fail when 'config.yml' is not present"
+  ansible.builtin.fail:
+    msg: "You need to setup your config file. link_to_docs here"
+  when: (not tqm_config.stat.exists) and (not continuous_integration)
```

Comment:
I'm assuming that this is the reason its skipping it? I haven't looked too much into this as I'm studying but. If I remove this CI entry on line 61, would it not skip this action? or why would it think there's a config otherwise?